shared/src/packages/sites/libs/types/site-get-by-project-request-dto.type.ts

```diff
@@ -0,0 +1,5 @@
+type SiteGetByProjectRequestDtoType = {
```

Comment:
```suggestion
type SiteGetByProjectParamsDto = {
```
don't forget about file name